### PR TITLE
feat: Create anchor commits directly on ipfs

### PR DESCRIPTION
**Pull request by stbrody**

### CODE REVIEW

**Review by v-stickykeys — APPROVED** [review]
LGTM

**Review by v-stickykeys — APPROVED** [review]
👍

File: src/services/anchor-service.ts [hunk 1]
```diff
@@ -306,10 +306,12 @@ export default class AnchorService {
...
-      const anchorCid = await this.ceramicService.publishAnchorCommit(
-        candidate.streamId,
-        ipfsAnchorCommit
-      )
+      // TODO(#548): Publish anchor commits via Ceramic
+      // const anchorCid = await this.ceramicService.publishAnchorCommit(
+      //   candidate.streamId,
+      //   ipfsAnchorCommit
+      // )
+      const anchorCid = await this.ipfsService.storeRecord(ipfsAnchorCommit)
```
Comment (oed) [inline src/services/anchor-service.ts:314]: Why do we need to store the anchor commit? This commit would be created by js-ceramic nodes when they get the anchored successful response?

**Review by oed — APPROVED** [review]
Not sure why we need to store the anchor commit itself into ipfs, but I guess it doesn't really hurt.

File: src/services/anchor-service.ts (hunk 1 above)
Comment (stbrody) [reply src/services/anchor-service.ts:314]: The CAS just passes the anchor commit CID to the ceramic node that requested the anchor. The ceramic node then loads the anchor commit from ipfs, applies it, and publishes it to pubsub.

If the commit doesn't exist on ipfs then loading the commit on ceramic will fail

Comment (oed) [reply src/services/anchor-service.ts:314]: Oh I see.

**Pull request merged**